### PR TITLE
Send helics-version-update event to helics-packaging after new releases

### PR DESCRIPTION
**Pull request by nightlark**
### Summary
If merged this pull request will send an event after all HELICS release binaries are built to helics-packaging telling it that a new version has been released.

The action that handles the event in `helics-packaging` updates the version file, but doesn't open a PR yet -- it shows a [git diff](https://github.com/GMLC-TDC/helics-packaging/runs/483558701?check_suite_focus=true#step:4:10) of the change that would be in a PR.

### Proposed changes
- Send a `helics-version-update` event to `helics-packaging` after the the release build is finished

### CODE REVIEW

File: .github/workflows/release-build.yml [hunk 1]
```diff
@@ -7,9 +7,9 @@ on:
     types: published
 
 jobs:
-################################
+#####################################
```
Comment (phlptp) [inline .github/workflows/release-build.yml]: Is there a reason all these are longer?

Comment (nightlark) [reply .github/workflows/release-build.yml]: Just to match the length of the longest header name

File: .github/workflows/release-build.yml [hunk 2]
```diff
@@ -291,3 +291,21 @@ jobs:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         UPLOAD_URL: ${{ github.event.release.upload_url }}
       run: ./.github/workflows/upload-release-asset.sh "artifacts/Helics-$(git rev-parse --abbrev-ref "${GITHUB_REF}")-SHA-256.txt"
+
```
Comment (phlptp) [inline .github/workflows/release-build.yml]: Is there a corresponding receiver in the other repos?

Comment (nightlark) [reply .github/workflows/release-build.yml]: The git diff link in the PR description goes to one of them.

**Comment by phlptp** [comment]
This is related to #898 ?

**Comment by nightlark** [comment]
Indirectly - helics-packaging will be set up to send version update events to any other repositories that need updates.

**Pull request merged**